table.csv

```diff
@@ -175,6 +175,7 @@ kangarootwelve,                 multihash,      0x1d01,         draft,      Kang
 aes-gcm-256,                    encryption,     0x2000,         draft,      AES Galois/Counter Mode with 256-bit key and 12-byte IV
 silverpine,                     multiaddr,      0x3f42,         draft,      Experimental QUIC over yggdrasil and ironwood routing protocol
 sm3-256,                        multihash,      0x534d,         draft,
+sha256a,                        multihash,      0x7012,         draft,      The sum of multiple sha2-256 hashes
```

Comment:
```suggestion
sha256a,                        multihash,      0x7012,         draft,      The sum of multiple sha2-256 hashes; as specified by Ceramic CIP-124.
```
I know the "Ceramic" is redundant with the "C" in "CIP", but my hope would that this way it's easy for people to find the spec.

Reply:
Thanks, wasn't sure about convention about adding information on where to find a spec.

Reply:
There isn't really a convention. I just looked what we did in other cases :)